vision-quickstart/app/src/main/java/com/google/mlkit/vision/demo/VisionProcessorBase.java

```diff
@@ -80,6 +83,8 @@
     private ByteBuffer processingImage;
     @GuardedBy("this")
     private FrameMetadata processingMetaData;
+    private ArrayList<Pair<Image, Long>> bufferImage = new ArrayList<Pair<Image, Long>>();
```

Comment:
Can you name this something that makes it clear that it is a collection, eg bufferImageList?  This makes it much easier to read the code later on, I keep thinking it is a single Image.